project/Dependencies.scala

```diff
@@ -3,35 +3,36 @@ import sbt._
 object Dependencies {
   val akkaV = "2.5.32"
   val akkaHttpV = "10.2.9"
-  val jacksonV = "2.12.2"
-  val jacksonHotfixV = "2.12.2" // for when only some of the Jackson libs have hotfix releases
-  val nettyV = "4.1.74.Final"
+  val jacksonV = "2.13.2"
+  val jacksonHotfixV = "2.13.2.2" // for when only some of the Jackson libs have hotfix releases
+  val nettyV = "4.1.77.Final"
+  val workbenchLibsHash = "2c48b00"
 
   def excludeGuava(m: ModuleID): ModuleID = m.exclude("com.google.guava", "guava")
   val excludeAkkaActor =        ExclusionRule(organization = "com.typesafe.akka", name = "akka-actor_2.13")
   val excludeAkkaStream =       ExclusionRule(organization = "com.typesafe.akka", name = "akka-stream_2.13")
   val excludeAkkaHttp = ExclusionRule(organization = "com.typesafe.akka", name = "akka-http_2.13")
   val excludeSprayJson = ExclusionRule(organization = "com.typesafe.akka", name = "akka-http-spray-json_2.13")
 
-  val rootDependencies = Seq(
+  val rootDependencies: Seq[ModuleID] = Seq(
     // proactively pull in latest versions of these libraries, instead of relying on the versions
     // specified as transitive dependencies, due to OWASP DependencyCheck warnings for earlier versions.
+    // TODO: can these move to sbt's dependencyOverrides?
     "com.fasterxml.jackson.core"     % "jackson-annotations" % jacksonV,
     "com.fasterxml.jackson.core"     % "jackson-databind"    % jacksonHotfixV,
     "com.fasterxml.jackson.core"     % "jackson-core"        % jacksonV,
-    "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % jacksonV,
```

Comment:
this isn't necessary. It doesn't show up as a transitive dependency of anything else, and removing it doesn't cause problems. It's possible one of the other dependency upgrades removed the need for this.